section1/reflection.md

```diff
@@ -1,19 +1,56 @@
 
-3. How would you log the string `"Hello World!"` to the console?
+2. What are the data types you learned about in this section? In your own words, define each.  
 
-4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
+   The **Number** data type, which is used in JS to handle all numbers, including negative and floating-point numbers.  
 
-5. In your own words, what is a variable? How would you explain it to a 5 year old?
+   The **String** data type, which is used in JS to handle any kind of text.  
 
-6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
+   The **Boolean** data type, which is used in JS to evaluate whether a value is *true* or *false*.
 
-7. In your own words, explain what concatenation is.
+3. How would you log the string `"Hello World!"` to the console?  
 
-8. Think of a site or app you use frequently. Where do you think the developers used concatenation?
+   I would use the code below:  
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+```javascript
+var greeting = "Hello World!";
+console.log(greeting);
+```
+
+4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?  
+
+   The character(s) I would use to indicate comments in a JS file are `//` and `/* */`.  
+
+   Code comments are used to communicate to other developers about what my code is doing.  
+   Another purpose of code commenting is to temporarily deactivate blocks of code to keep them from running.  
+
+5. In your own words, what is a variable? How would you explain it to a 5 year old?  
+
+   A *variable* is used in JS to represent and store some sort of value.  
+
+   A variable is like a box, which can be used to store valuable things that you may want to use later.
+
+6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?  
+
+   Most sites that I visit usually ask for **Username**, **Email** and **Password**.  
+   So the variables are probably `userName`, `userEmail` and `userPassword`.  
+   I believe each of those variables would hold the *string* data type.
+
+7. In your own words, explain what concatenation is.  
+
+   *Concatenation* is the process of combining multiple values together.
+
+
+8. Think of a site or app you use frequently. Where do you think the developers used concatenation?  
+
+   I think many sites use *Concatenation* to combine my user-name with their greeting message.
+
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)  
+
+   I'm sure I can (and I will) google this. But I'm wondering, how are programming languages like JS created?  
```

Comment:
HA YES so interesting. I've gone down this rabbit hole. You've probably already googled it so I won't answer but in case you want to nerd out about something else, I think interpreted vs compiled languages is super interesting.